rllib/agents/trainer.py

```diff
@@ -683,6 +684,9 @@ def log_result(self, result: ResultDict):
 
     @override(Trainable)
     def setup(self, config: PartialTrainerConfigDict):
+        update_global_seed_if_necessary(
```

Comment:
if we do this here, we shouldn't have to call this again in the env class right?
feel like this is a much better place to condition these global modules.

Reply:
We still have to do this for all remote workers, but maybe not for worker-index=0 (the "local worker"). I can add an if-block in rollout_worker.py ...

Reply:
Done.

Reply:
that's right. gotta change my mindset a little bit.
things like this may be the reason sometimes people say putting everything in a single file is a confusing.